back/appsync/mapping-templates/getIdea-response-mapping-template.txt

```diff
@@ -1 +1,5 @@
-$util.toJson($context.result.items[0])
+#if($context.result.items.size() > 0)
+  $util.toJson($context.result.items[0])
+#else
+  null
```

Comment:
returning null will be not json and will break frontend i suppose

Reply:
In any case there is try catch, it will return 404, it will not break anything.

Reply:
But still why would be want to return not valid json from backend if we work via json? If its 200 request it should be json

Reply:
What data to put in json if object is not found?

Reply:
$utils.error if we want error, but maybe just empty object

Reply:
Ok, added error